yolact.py

```diff
@@ -564,20 +235,21 @@ def freeze_bn(self, enable=False):
     def forward(self, x):
         """ The input should be of size [batch_size, 3, img_h, img_w] """
         _, _, img_h, img_w = x.size()
+        cfg = self.cfg
         cfg._tmp_img_h = img_h
```

Comment:
I sometimes use this "hack" instead of changing the whole block from `cfg` to `self.cfg`, is that ok, style-wise?

Reply:
Should be okay, since self.x is always so ugly (I really wish Python worked like other OOP languages in that regard).

Reply:
Though it looks like you use both cfg and self.cfg in the same function here. I'd pick one and stick with it.